plugins/versions.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const checker = require('license-checker');
+const process = require('process');
+const VError = require('verror');
+
+/**
+ * Hapi interface for plugin to return package list
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Object}         options
+ * @param  {Function} next
+ */
+exports.register = (server, options, next) => {
+    const SD_REGEX = /^screwdriver-/;
+
+    // Load licenses
+    checker.init({
+        production: true,
+        start: process.cwd()
+    }, (err, json) => {
+        if (err) {
+            return next(new VError(err, 'Unable to load package dependencies'));
+        }
+
+        const depArray = Object.keys(json).map(key => Object.assign({ name: key }, json[key]));
+        const otherDeps = depArray.map(dep => ({
+            name: dep.name.split('@').slice(0, -1).join('@'),
+            repository: dep.repository || 'UNKNOWN',
+            licenses: dep.licenses || 'UNKNOWN'
+        }));
+        const sdDeps = depArray.filter(dep => SD_REGEX.test(dep.name)).map(dep => dep.name);
+
+        server.route({
+            method: 'GET',
+            path: '/versions',
+            handler: (request, reply) => reply({
+                versions: sdDeps,
```

Comment:
I think `other-dependences` will be too much, just look at how big it is in the build :(